server/server.js

```diff
@@ -18,11 +18,6 @@ app.use(cookieParser());
 
 app.use(bodyParser.json());
 
-app.use((req, res, next) => {
-  console.log(req.cookies);
-  next();
-});
-
 app.get('/api/users', (req, res) => {
   getUsers()
     .then(users => res.json(users))
```

Comment:
Is the data in getUser() should be displalyed only for logged in users?, if so, should check the cookies before responding to the client with data

Reply:
No, it's not sensitive. It's used to display the table.

Reply:
isn't the table is in the prediction page which only logged in user should see ?

Reply:
Aha I see so you don't have any requests that needs the user to be logged in yet beside `api/cookieValidation`